hb_report/utillib.py

```diff
@@ -1439,6 +1440,25 @@ def say_ssh_user():
         return constants.SSH_USER
 
 
+def sbd_info():
+    """
+    sbd information
+    """
+    out_string = "#####SBD info:\n\n"
+    sbd_device = grep("(^|^\s*)SBD_DEVICE=.*", infile=constants.SBDCONF)
+    if sbd_device:
+        device = sbd_device[0].split('=')[-1]
+        for cmd in ["dump", "list"]:
+            out_string += "## sbd {}:\n".format(cmd)
+            out_string += get_command_info("sbd -d {} {}".format(device, cmd))[1] + '\n'
+        shutil.copy2(constants.SBDCONF, constants.WORKDIR)
```

Comment:
just save the conf file if it exists. since sbd-1.3.0 or later supports diskless mode

Reply:
Yes, I am not sure dumping the actual SBD device is a good idea for crm report... I would suggest to just save the configuration file in the report.